semconv/src/main/java/io/opentelemetry/semconv/trace/attributes/SemanticAttributes.java

```diff
@@ -7,350 +7,14 @@
 
 import static io.opentelemetry.api.common.AttributeKey.booleanKey;
 import static io.opentelemetry.api.common.AttributeKey.longKey;
-import static io.opentelemetry.api.common.AttributeKey.stringArrayKey;
 import static io.opentelemetry.api.common.AttributeKey.stringKey;
 
 import io.opentelemetry.api.common.AttributeKey;
-import java.util.List;
 
 // DO NOT EDIT, this is an Auto-generated file from
-// buildscripts/semantic-convention//templates/SemanticAttributes.java.j2
+// buildscripts/semantic-convention/templates/SemanticAttributes.java.j2
 public final class SemanticAttributes {
 
-  /** Name of the cloud provider. */
-  public static final AttributeKey<String> CLOUD_PROVIDER = stringKey("cloud.provider");
```

Comment:
my only question: do we need to deprecate each and every one of these that were in the wrong place?

Reply:
If follow the rules yes, but we can be a bit more relaxed here

Reply:
If these stay with deprecation, I would be able to deflect the pain of these from when I update the instrumentation repo to you doing it after, presumably you don't want that :P

Reply:
I'll work on deprecating these. I might be able to figure out how to generate that automatically.

Reply:
ok, deprecated. I did it manually since there didn't seem to be an obvious way to do it programmatically in the template.